ErgoScript-Developer-Course/batch-0/Individual/Mx62/hw2/hw2.scala

```diff
@@ -0,0 +1,9 @@
+val testBool = true && false
+val testLong = if (testBool) 1 else 0
+println("testLong: " + testLong)
+val testColl = Coll(1, 2, 3)
+def testPair = (0, 2)
```

Comment:
Your hw is fine, just try to include the data types when writing professional code, as it makes it easier for the reader and yourself when trying to find errors.